docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  platform-frontend:
+    image: us.gcr.io/dbet-platform/platform-fronend:latest
```

Comment:
Shouldn't this be `us.gcr.io/dbet-platform/platform-frontend:latest`

Reply:
fixed